content/en/news/releases/1.16.x/announcing-1.16.3/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Announcing Istio 1.16.3
+linktitle: 1.16.3
+subtitle: Patch Release
+description: Istio 1.16.3 patch release.
+publishdate: 2023-02-21
```

Comment:
I would recommend adding a time to the date, so something like: `2023-02-21T08:00:00-06:00` and changing the time so that the latest time is the 1.17, and earliest is the 1.15. Since these are part of different PRs it's not easy to tell if these will sort in the same order or not.

Reply:
Done.